packages/cosmwasm/src/lcdapi/wasm.spec.ts

```diff
@@ -430,6 +430,56 @@ describe("wasm", () => {
       expect(await client.wasm.getContractInfo(nonExistentAddress)).toBeNull();
     });
 
+    it("can list contract history", async () => {
+      pendingWithoutWasmd();
+      const wallet = await Secp256k1Wallet.fromMnemonic(alice.mnemonic);
+      const client = makeWasmClient(wasmd.endpoint);
+      const beneficiaryAddress = makeRandomAddress();
+      const transferAmount: readonly Coin[] = [
```

Comment:
This can be written more compact as ` const transferAmount = coins(707707, "ucosm")`